crates/pop-cli/src/common/build.rs

```diff
@@ -14,9 +14,13 @@ pub fn has_contract_been_built(path: Option<&Path>) -> bool {
 		Ok(manifest) => manifest,
 		Err(_) => return false,
 	};
-	let contract_name = manifest.package().name();
-	project_path.join("target/ink").exists() &&
-		project_path.join(format!("target/ink/{}.contract", contract_name)).exists()
+	if manifest.package.as_ref().is_none() {
```

Comment:
As mentioned in this comment: https://github.com/r0gue-io/pop-cli/pull/347/files#r1854713654

Reply:
Can simplify to use 

```
manifest.package.map(|p| { 
  project_path.join(format!("target/ink/{}.contract", p.name())).exists()
 })
.unwrap_or_default()
```